components/screens/TokenAsset.tsx

```diff
@@ -0,0 +1,51 @@
+import Header from "@/components/GeneralHeader.tsx";
+import { useNavigate, useParams } from "react-router-dom";
+import { twMerge } from "tailwind-merge";
+import React, { useState } from "react";
+import TokenHistory from "@/components/token-asset/TokenHistory.tsx";
+import TokenInfo from "@/components/token-asset/TokenInfo.tsx";
+
+export default function TokenAsset() {
+  const navigate = useNavigate();
+  const { ticker } = useParams();
+  const [activeTab, setActiveTab] = useState<"history" | "info">("history");
+
```

Comment:
It would be better to cache the data from API:
![Uploading 2025-02-10 18-56-40.gif…]()

Reply:
23c79e872e35c519e6cc2b06e5921faf00be7d79

SWR to save the day again ;).